tools/run-task.sh

```diff
@@ -5,6 +5,9 @@
 
 set -eu
 
+trap "exit" INT TERM
+trap "kill -- -$$" EXIT
```

Comment:
I needed [StackOverflow](https://superuser.com/a/562804) to understand this.

Reply:
I needed stackoverflow to work out how to do it 🙈 

Shame there's not a more obvious way of doing it, but y'know... shell-scripts.

At some point this thing either gets wrapped up as a gds-cli command or replaced with nicer AWS tools, so it's only temporary.